### DISCUSSION TITLE
Deleting a forked repository closes my pull requests

### DISCUSSION THREAD

**Discussion by lnedry** [haraka/Haraka#3233] [General]
I submitted several pull requests on Sep 9. I just checked on them and I see that they were closed by me an hour after I submitted them. Github says, "lnedry closed this by deleting the head repository". Do I need to resubmit all of those PRs and this time keep my fork around until the changes have been merged?

**Comment by msimerson** [comment]
Apparently so. The typical workflow is:

Create a branch, make changes, submit PR, after PR is merged delete branch.
